pycpt/src/pycpt/notebook.py

```diff
@@ -637,7 +637,12 @@ def plot_forecasts(
     files_root,
     predictor_names,
     MOS,
+    vmin=None,
+    vmax=None,
 ):
+    if predictand_name is None:
+        predictand_name='PRCP'
+
```

Comment:
I'd like to know what problem this is fixing. I haven't needed to make these changes to notebook.py on my branch.

Reply:
As we disussed in person, please leave the vmin and vmax change in, but undo the predictand_name change. Notebook should define `predictand_name = SOMETHING.PRCP`.